app/src/main/java/org/fossasia/openevent/general/auth/LoginFragment.kt

```diff
@@ -108,10 +108,13 @@ class LoginFragment : Fragment() {
 
     private fun redirectToMain(bundle: Bundle?) {
         val intent = Intent(activity, MainActivity::class.java)
-        if (((bundle != null) && !id.equals(-1)) && (ticketIdAndQty != null)) {
-            intent.putExtra(LAUNCH_ATTENDEE, true)
+        if (bundle != null) {
+            if (!id.equals(-1) && ticketIdAndQty != null) {
+                intent.putExtra(LAUNCH_ATTENDEE, true)
+            }
             intent.putExtras(bundle)
         }
+
```

Comment:
Extra feed

Reply:
Sorry about that, I will take care of these by next time.

Reply:
> Extra feed

Sorry about that, I will take care of these by next time.